src/in_range.rs

```diff
@@ -71,11 +71,13 @@ macro_rules! impl_const_new {
         }
 
         impl<const MIN: u128, const MAX: u128> InRange<$T, MIN, MAX> {
+            #[allow(unused)]
```

Comment:
I thought these are used.  Or is it just that they're unused for some `$T`?

Reply:
Most `T` are unused.